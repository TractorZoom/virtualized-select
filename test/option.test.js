import React from 'react'
import Option from '../src/option'
import renderer from 'react-test-renderer'

test('Option with no subtitle', () => {
    const component = renderer.create(<Option label='some label' value='some value' />)
    let tree = component.toJSON()

    expect(tree).toMatchSnapshot()
})

test('Option with subtitle', () => {
    const component = renderer.create(<Option label='some label' subtitle='some subtitle' value='some value' />)
    let tree = component.toJSON()

    expect(tree).toMatchSnapshot()
})
