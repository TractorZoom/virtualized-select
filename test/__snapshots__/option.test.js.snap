// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Option with no subtitle 1`] = `
<div
  className="option-container"
>
  <div>
    some label
  </div>
</div>
`;

exports[`Option with subtitle 1`] = `
<div
  className="option-container"
>
  <div>
    some label
  </div>
  <div
    className="option-subtitle"
  >
    <span>
      some subtitle
    </span>
  </div>
</div>
`;
